.github/workflows/genesis-diff.yml

```
@@ -33,7 +33,48 @@ jobs:
 
     - name: compute diff
       working-directory: validation
-      id: run-command
+      id: run_command
       run: |
         ./scripts/diff-configs.sh ./generate-genesis/output-base ./generate-genesis/output-HEAD
+      continue-on-error: true
 
+    - name: Check command status and comment on PR
+      if: steps.run_command.outcome == 'failure'
+      uses: actions/github-script@v6
+      with:
+        github-token: ${{ secrets.GITHUB_TOKEN }}
+        script: |
+          const issue_number = context.payload.pull_request.number;
+          const run_id = context.runId;
+          const repo = context.repo.repo;
+          const owner = context.repo.owner;
+          const run_url = `https://github.com/${owner}/${repo}/actions/runs/${run_id}`;
+          const body = `Detected nonzero diff in genesis. This PR modifies the genesis data of at least one chain in the registry, which may or may not have been intended. It could affect whether a node started up with this data can successfully sync with the wider network. You can see the detail [here](${run_url}).`;
```

```suggestion
          const body = `Detected nonzero diff in genesis. This PR modifies the genesis data of at least one chain in the registry, which may or may not have been intended. You can see the detail [here](${run_url}).`;
```
I would remove that one sentence, a consumer of this bot message probably doesn't need an explanation here, so better to keep it short.

---

[b92e4ef](https://github.com/ethereum-optimism/superchain-registry/pull/447/commits/b92e4ef81b6002f51cdc20f1140fa6553b931f8b)